catalog/src/main/resources/db/migration/V13__Add_a_row_to_dssets.sql

```diff
@@ -0,0 +1 @@
+insert into ds_sets(name, default_datasource_id) values('quark_default', null);
```

Comment:
Eventually when both them use the same copy, it will applied. It will be harmless since it will add a dummy row that we will not use.